changelog/changes/2023-10-19-torcx-removal.md

```diff
@@ -0,0 +1 @@
+- torcx was replaced by systemd-sysext in the OS image. Learn more about sysext and how to customise OS images [here](https://www.flatcar.org/docs/latest/provisioning/sysext/). `update_engine` and the `kola` test suite in Mantle have been adapted to this change accordingly. ([scripts#1216](https://github.com/flatcar/scripts/pull/1216), [update_engine#30](https://github.com/flatcar/update_engine/pull/30), [Mantle#466](https://github.com/flatcar/mantle/pull/466), [Mantle#465](https://github.com/flatcar/mantle/pull/465)).
```

Comment:
```suggestion
- Torcx, the mechanism to switch Docker versions, was replaced systemd-sysext in the OS image. Learn more about sysext and building your own OS extension images [here](https://www.flatcar.org/docs/latest/provisioning/sysext/). `update_engine` and the `kola` test suite in Mantle have been adapted to this change accordingly. ([scripts#1216](https://github.com/flatcar/scripts/pull/1216), [update_engine#30](https://github.com/flatcar/update_engine/pull/30), [Mantle#466](https://github.com/flatcar/mantle/pull/466), [Mantle#465](https://github.com/flatcar/mantle/pull/465)).
```

Reply:
I still think that the Mantle/update_engine changes don't belong in the end-user release notes because they are internal

Reply:
Regularly I would agree - but in this case we're introducing significant changes. I just want to make sure we leave a paper trail and we make sufficient noise around it.

Reply:
Then let's add more words around the change. The mantle/update_engine mentions are completely irrelevant to users which could rather mean that they skip reading the whole entry because it just looks like noise to them.

Reply:
E.g., `You can convert existing Torcx images with the helper script, use the published systemd-sysext images for Docker/Kubernetes from the sysext-bakery, or build your own sysext images for particular Docker versions.`

Reply:
I've elaborated more and added sub-bullet-points to the changelog entry. This isn't really what you asked but I hope it makes it more interesting for users to read, and also adds the pointers we've discussed here in a more user friendly way.

Reply:
Thanks, that's much better :)